contracts/0.8.25/vaults/ValidatorConsolidationRequests.sol

```diff
@@ -80,68 +65,102 @@ contract ValidatorConsolidationRequests {
      *      | ----- public key (48 bytes) ----- || ----- public key (48 bytes) ----- | ...
      *
      * @param _refundRecipient The address to refund the excess consolidation fee to.
-     * @param _stakingVault The address of the staking vault contract.
+     * @param _dashboard The address of the dashboard contract.
      * @param _adjustmentIncrease The sum of the balances of the source validators to increase the rewards adjustment by.
```

Comment:
probably we should emphasize importance of this param (now another method contains this explanation, but this one is not, and we need to articulate fee change)